framework/quota/src/main/java/org/apache/cloudstack/quota/dao/QuotaEmailTemplatesDaoImpl.java

```diff
@@ -68,4 +68,14 @@ public Boolean doInTransaction(final TransactionStatus status) {
             }
         });
     }
+
+    @Override
+    public QuotaEmailTemplatesVO findById(long id) {
+        return Transaction.execute(TransactionLegacy.USAGE_DB, new TransactionCallback<QuotaEmailTemplatesVO>() {
+            @Override
+            public QuotaEmailTemplatesVO doInTransaction(final TransactionStatus status) {
+                return QuotaEmailTemplatesDaoImpl.super.findById(id);
+            }
+        });
```

Comment:
IMO, we could increment readability here with lambda.

Reply:
Why was this marked as resolved? It is not using lambda.